.evg.yml

```diff
@@ -163,8 +279,8 @@ functions:
         aws_session_token: ${AWS_SESSION_TOKEN}
         local_files_include_filter:
```

Comment:
Is it possible to use ${SBOM_LITE_NAME} to make sure that the file name is the same throughout?